v2/tools/generator/internal/codegen/pipeline/add_simple_cross_resource_references.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+)
+
+// AddSimpleCrossResourceReferencesStageID is the unique identifier for this pipeline stage
+const AddSimpleCrossResourceReferencesStageID = "addsimpleCrossResourceReferences"
```

Comment:
I don't see the link between what this stage does and what it's called.

Reply:
Changed this name (and the name of the function)